.travis.yml

```diff
@@ -4,9 +4,9 @@ services:
   - docker
 env:
   global:
-    - K8S_VERSION=v1.13.4
-    - MINIKUBE_VERSION=v0.35.0
-    - HELM_VERSION=v2.13.0
+    - K8S_VERSION=v1.18.0
+    - MINIKUBE_VERSION=v1.11.0
```

Comment:
Same here.. there is sooo many versions in between, it's scary :fearful: 
1.0 was the next version after 0.35.0.

I can see in their changelog that 1.4.0 was the first release to support k8s 1.16.0 and that maybe v1.5.2 could be a good pick? Who knows. Wanna try?

Reply:
Let's go 1.12!!!!!